src/glang/lib/std/wrappers.c3

```diff
@@ -92,11 +114,11 @@ function [T] has_value : (self : Optional<T&> mut&) -> bool = {
 
 // HUGE DUPLICATION FOR MUTABLE REFERENCE
 // Optional<T&>
-function [T] make<T&> : (value : T mut&) -> Optional<T mut&> = {
-    return { ref_to_addr(&value) };
+function [T] make<Optional<T mut&>> : () -> Optional<T mut&> = {
+    return { 0 };
 }
 
-function [T] make<T&> : (value : T mut&) -> Optional<T mut&> = {
+function [T] make<T mut&> : (value : T mut&) -> Optional<T mut&> = {
     return { ref_to_addr(&value) };
 }
```

Comment:
This has the same bug you fixed in the immutable case eg. it should be `make<Optional<T mut&>>`

Reply:
We also have this in the immutable versions
```
function [T] make<T&> : (value : T&) -> Optional<T&> = {
    return { ref_to_addr(&value) };
}
```

I'll change this too then